datasets/ar/repet/ar_repet.yml

```diff
@@ -110,3 +110,17 @@ lookups:
           - null
           - Unknown
         value: null
+  type.name:
+    lowercase: true
+    options:
+      - match: Hay’at Tahrir al-Sham (HTS) (هيئة تحرير الشام (original script); Hay’at Tahrir al-Sham; Hay’et Tahrir al-Sham; Hayat Tahrir al-Sham; Assembly for the Liberation of Syria; Assembly for the Liberation of the Levant; Liberation of al-Sham Commission; Liberation of the Levant Organisation Tahrir al-Sham; Tahrir al-Sham Hay’at)
+        values: 
+          - Hay’at Tahrir al-Sham
+          - هيئة تحرير الشام
+          - Hay’et Tahrir al-Sham
+          - Hayat Tahrir al-Sham
+          - Assembly for the Liberation of Syria
+          - Assembly for the Liberation of the Levant
+          - Liberation of al-Sham Commission
+          - Liberation of the Levant Organisation Tahrir al-Sham
+          - Tahrir al-Sham Hay’at
```

Comment:
could you see if a "multisplit(val, ["original script", ";"]) gives a sane result for other affected entities? I found the entities.delta.json useful for this but I'm not sure if you can have that generated yet. See also https://github.com/opensanctions/opensanctions/pull/1154/files#diff-9d298192148def6763087fd465daebdafec284a907e20697b6a51f349f3a4ca1R11